### ISSUE TITLE
LocalVideoTrack.createCameraTrack() is not releasing camera notification status

### ISSUE THREAD

**Issue by FreddyKaiser** [livekit/client-sdk-flutter#178]
I'm using `LocalVideoTrack.createCameraTrack()` to display a preview before joining the conferencing room to the user.

When connecting to the Room over the fastConnectOptions I provide this preview track as `TrackOption()`
````
_room.connect(
        roomOptions: RoomOptions(adaptiveStream: true, dynacast: true, stopLocalTrackOnUnpublish: true),
        fastConnectOptions: FastConnectOptions(
          microphone: TrackOption(enabled: joinWithMic),
          camera: TrackOption(enabled: joinWithVideo, track: preview),
        ),

)
````

This seems working as desired and when leaving the conference room over .disconnect(), the Camera is no more indicated to be used by the App.

But when not calling the .connect on Preview and leaving the Join Dialog, the Camera seems not to be released as indicated by the system notification icon.
![NotReleasingCamera](https://user-images.githubusercontent.com/4628116/192545171-5b864cda-7c09-4bb9-9bf3-e39f8177ba31.jpg)

Also visible from the Flutter logs:
````
I/org.webrtc.Logging(19597): CameraStatistics: Camera fps: 30.
I/org.webrtc.Logging(19597): CameraStatistics: Camera fps: 30.
I/org.webrtc.Logging(19597): CameraStatistics: Camera fps: 30.
I/org.webrtc.Logging(19597): CameraStatistics: Camera fps: 30.
I/org.webrtc.Logging(19597): CameraStatistics: Camera fps: 30.
I/org.webrtc.Logging(19597): CameraStatistics: Camera fps: 31.
I/org.webrtc.Logging(19597): CameraStatistics: Camera fps: 30.
````

I tried several options on disposal of the Join Dialog as LocalTrack.dispose(), .disable(), .stop()... but nothing seems working properly. Any suggestions?

**Comment by cloudwebrtc** [comment]
I believe the issue is related to issue #166

**Comment by FreddyKaiser** [comment]
> I believe the issue is related to issue #166
Could be, interesting is that 166, at least for me, only affects Anrdoid and not iOS

**Comment by FreddyKaiser** [comment]
Even with `livekit_client: 1.1.5`and the related fix #166 this issue remains

**Comment by cloudwebrtc** [comment]
Should be fixed by this PR, I'm verifying
https://github.com/flutter-webrtc/flutter-webrtc/pull/1103

**Comment by cloudwebrtc** [comment]
@FreddyKaiser Should be fixed in this PR #188, it is worth noting that if `room.connect` fails, the ownership of the `preview` will be returned to you and you have to release it yourself, and `room.connect` succeeds will release the `preview` track in `room.disconnect`.

**Comment by FreddyKaiser** [comment]
> you have to release it yourself

I tried several different calls in order to release `previewVideo = await LocalVideoTrack.createCameraTrack();`
nothing works:
````
await previewVideo.mute();
await previewVideo.stop();
await previewVideo.disable();
await previewVideo.dispose();
````
and the notification status remains active. Any hints?
